### PR TITLE
Add Apache DataFu

### PR DESCRIPTION
**Pull request by eyala**
I used the template, let me know if anything is wrong. I know there's only one issue listed now but more are on their way, I promise!

### CODE REVIEW

File: datafu.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- pig
+- java
+- python
+- spark
+- hadoop
+- python
+- hive
+upforgrabs:
+  name: up for grabs
+  link: https://issues.apache.org/jira/issues/?jql=project%20%3D%20datafu%20AND%20status%20%3D%20Open%20AND%20labels%20%3D%20up-for-grabs%20ORDER%20BY%20created%20DESC
```
Comment (ritwik12) [inline datafu.yml]: @eyala This seems good too but we generate projects stats. So how about using `https://github.com/apache/datafu/labels/up%20for%20grabs`?
You will have to create a label `up for grabs` under that Github repo first.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: datafu.yml (hunk 1 above)
Comment (eyala) [reply datafu.yml]: The thing is, we put bugs and small feature requests in our JIRA, not on GitHub. Is it possible to add more than one link?

Comment (ritwik12) [reply datafu.yml]: Nope, in that case I would keep this link. But then you won't be able to get projects stats on the site like other projects.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@eyala Please move your file to `_data/projects/`.

**Comment by ritwik12** [comment]
@eyala Thanks!!

**Pull request merged**